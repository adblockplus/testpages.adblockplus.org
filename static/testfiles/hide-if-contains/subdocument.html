<head>
  <style>
  body {
    margin: 0 auto 0 auto;
    padding: 0;
    font-size: 16px;
    background-color: #f2f2f2;
    width: 1000px;
    max-width: 100%;
  }

  [data-expectedresult="fail"] {
    padding: 0.2em;
    color: #fff;
    background-color: #c70d2c;
  }

  .testcase-examplecontent {
    padding: 0.5em;
  }
  </style>
</head>

<body>
  <div class="testcase-examplecontent">Frame Content</div>
  <span id="frame-target" data-expectedresult="fail">This text should be hidden</span>
</body>
