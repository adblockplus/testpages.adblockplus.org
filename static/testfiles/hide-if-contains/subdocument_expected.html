<head>
  <style>
  body {
    margin: 0 auto 0 auto;
    padding: 0;
    font-size: 16px;
    background-color: #f2f2f2;
    width: 1000px;
    max-width: 100%;
  }

  .testcase-examplecontent {
    padding: 0.5em;
  }
  </style>
</head>

<body>
  <div class="testcase-examplecontent">Frame Content</div>
</body>
